hbase-server/src/main/java/org/apache/hadoop/hbase/master/snapshot/TakeSnapshotHandler.java

```diff
@@ -129,6 +132,8 @@ public TakeSnapshotHandler(SnapshotDescription snapshot, final MasterServices ma
       "Taking " + snapshot.getType() + " snapshot on table: " + snapshotTable, false, true);
     this.snapshotManifest =
       SnapshotManifest.create(conf, rootFs, workingDir, snapshot, monitor, status);
+    this.lockAcquireTimeoutMs =
+      conf.getLong(HBASE_SNAPSHOT_MASTER_LOCK_ACQUIRE_TIMEOUT, 5 * 60 * 1000L);
```

Comment:
Would 5 minutes be a little long ? How about 60 seconds ? As I mentioned before, the client will time out after 60 seconds by default, so there seems to be no difference between waiting 60 seconds and waiting 5 minutes? What do you think?

Reply:
Sounds good, will make this change.

Reply:
done